pandas_gbq/gbq.py

```diff
@@ -162,6 +162,65 @@ class TableCreationError(ValueError):
     pass
 
 
+class Context(object):
+    """Storage for objects to be used throughout a session.
+
+    A Context object is initialized when the ``pandas_gbq`` module is
+    imported, and can be found at :attr:`pandas_gbq.context`.
+    """
+
+    def __init__(self):
+        self._credentials = None
+        self._project = None
+
+    @property
```

Comment:
Do we need the getters and setters? Are they doing anything beyond attributes? It's not bloating the interface, so I don't have a strong view.

Reply:
My thought is that these can be used to provide documentation about the settable properties.